src/os/bluestore/BlueStore.cc

```diff
@@ -3491,14 +3499,6 @@ void *BlueStore::MempoolThread::entry()
       next_balance = ceph_clock_now();
       next_balance += autotune_interval;
     }
-    if (resize_interval > 0 && next_resize < ceph_clock_now()) {
```

Comment:
This makes sense but will need to be updated slightly with the PriorityCacheManager.